Stonks/FinancialPlan.cs

```diff
@@ -74,5 +74,17 @@ public double GetSpendings()
             }
             return spendings;
         }
+
+        public double GetMaxExpense() {
+            double max_expense=0;
+
+            foreach (Expense expense in Expenses) {
+                if (expense.Value > max_expense) {
+                    max_expense = expense.Value;
+                }
+            }
+
+            return max_expense;
```

Comment:
Use camelCase for naming.
also can reduce code to one line
return Expenses.Max(x => x.Value);

Reply:
OK!